### PR TITLE
chore(deps): Including python-dotenv in requirements

### LINKED ISSUE #6236: Include Python-dotenv in our Requirements
#### Description:

Include Python-dotenv in our Requirements to remove the generated warnings indicating the presence of .env files.

#### Stacktrace:

```py
Users/kush/open-event-server/venv/lib/python3.7/site-packages/flask_limiter/extension.py:113: UserWarning: Use of the default `get_ipaddr` function is discouraged. Please refer to https://flask-limiter.readthedocs.org/#rate-limit-domain for the recommended configuration
  " for the recommended configuration", UserWarning
 * Tip: There are .env or .flaskenv files present. Do "pip install python-dotenv" to use them.
```

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #6236

### CODE REVIEW

**Comment by kushthedude** [comment]
@iamareebjamal Please Review

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
--r requirements/prod.txt
+-r requirements/prod.txt
```
Comment (mrsaicharan1) [inline requirements.txt]: Why was this change required?

**Comment by kushthedude** [comment]
I didnt change it,idk whats the change in the file

**Comment by kushthedude** [comment]
@iamareebjamal Please Review
